packages/devtools_app/lib/src/screen.dart

```diff
@@ -304,3 +308,20 @@ class BadgePainter extends CustomPainter {
     return true;
   }
 }
+
+class ShortcutsConfiguration {
+  ShortcutsConfiguration({
+    @required this.shortcuts,
+    @required this.actions,
+  });
+
+  final Map<ShortcutActivator, Intent> shortcuts;
+  final Map<Type, Action<Intent>> actions;
+
+  static ShortcutsConfiguration empty =
```

Comment:
make this a factory constructor

Reply:
Let me know what you think of it now - it's a factory constructor, and it's passed to the widget